airbyte-integrations/connectors/source-tmdb/source_tmdb/spec.yaml

```diff
@@ -0,0 +1,17 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Tmdb Spec
+  type: object
+  required:
+    - api_key
+    - movie_id
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      description: API Key
+      airbyte_secret: true
+    movie_id:
+      type: string
+      description: Target movie ID
```

Comment:
I don't think this is the best usage of the connector. What about using the search endpoint to retrieve all movies after a release date?

Reply:
Comment Resolved: Changed the flow of streams and made movie_id as optional.